### DISCUSSION TITLE
Custom tooltip for select

### DISCUSSION THREAD

**Discussion by mostopalove** [sa-si-dev/virtual-select#111] [Q&A] [answered]
How to add custom tooltip to select? Instead of using "value" as tooltip.

**Comment by sa-si-dev** [comment]
Could you please give me a sample scenario or example to understand your requirement.

**Reply by mostopalove** [reply]
I want to show user placeholder in case it croped by css text-ellipsis on small screen

**Reply by sa-si-dev** [reply]
I will add tooltip for cropped placeholder in the future release.

**Reply by mostopalove** [reply]
Thanks! I'm looking forward to it!

**Comment by sa-si-dev** [comment, marked as answer]
From the [release v1.0.27](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.27), it would show a tooltip if the placeholder is cropped.
